hugegraph-core/src/main/java/org/apache/hugegraph/util/StringEncoding.java

```diff
@@ -17,19 +17,22 @@
 
 package org.apache.hugegraph.util;
 
-import java.io.UnsupportedEncodingException;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+import java.io.IOException;
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
 import java.util.Base64;
 import java.util.UUID;
 
-import org.mindrot.jbcrypt.BCrypt;
-
 import org.apache.hugegraph.HugeException;
 import org.apache.hugegraph.backend.serializer.BytesBuffer;
+import org.mindrot.jbcrypt.BCrypt;
+
 import com.google.common.base.CharMatcher;
 
 /**
+ * TODO: lack license header & LICENSE reference
```

Comment:
https://github.com/JanusGraph/janusgraph/blob/master/janusgraph-core/src/main/java/org/janusgraph/util/encoding/StringEncoding.java
The janusgraph also is APV2 license, should i keep the janusgraph origin license? should i modify our license file?

Reply:
> [JanusGraph/janusgraph@`master`/janusgraph-core/src/main/java/org/janusgraph/util/encoding/StringEncoding.java](https://github.com/JanusGraph/janusgraph/blob/master/janusgraph-core/src/main/java/org/janusgraph/util/encoding/StringEncoding.java?rgh-link-date=2023-01-20T12%3A37%3A02Z) The janusgraph also is APV2 license, should i keep the janusgraph origin license? should i modify our license file?

same as the [previous one ](https://github.com/apache/incubator-hugegraph/pull/2095#discussion_r1098853204)